src/main/java/edu/tcu/cs/superfrogscheduler/user/UserRepository.java

```diff
@@ -10,4 +10,6 @@
 @Repository
 public interface UserRepository extends JpaRepository<SuperFrogUser, String>, JpaSpecificationExecutor<SuperFrogUser> {
     Optional<SuperFrogUser> findByEmail(String email);
+
+    Optional<SuperFrogUser> findById(String id);
```

Comment:
This seems unnecessary because the repo has built-in findById

Reply:
This was a mistake I will remove that